environs/config/config.go

```diff
@@ -973,6 +984,18 @@ func (c *Config) DefaultSeries() (string, bool) {
 	}
 }
 
+// SecretStore returns the secret store name.
+func (c *Config) SecretStore() string {
+	value, _ := c.defined[SecretStoreKey].(string)
+	return value
+}
+
+// SecretStoreConfig returns the secret store config (json or yaml).
```

Comment:
Nit I have a problem with this comment. We are returning a string. No validation has been done to assert that valid json or yaml exist here. We just have a blob of bytes.

Reply:
updated comment